packages/core/src/browser/frontend-application.ts

```diff
@@ -158,10 +160,10 @@ export class FrontendApplication {
     }
 
     /* vvv HOTFIX begin vvv
```

Comment:
/** to fix formatting, js-doc should belong to `inComposition`, right now its indentation is smaller

Reply:
Darn auto-format

Reply:
Fixed